### DISCUSSION TITLE
N64.txt missing

### DISCUSSION THREAD

**Discussion by rrig** [sanni/cartreader#202] [Need help]
When trying to read N64carts I get "N64.txt missing". I have attached the N64.txt file I am using. The only way I could find to get this file was to highlite and copy the raw file then paste to notepad. Would you know if this issue is a file issue or a hardware issue? (Game cartridge is Banjo Kazooie) I did not install the GBA slot as I only have SNES and N64. Thank You for any help you can provide.
[N64.txt](https://github.com/sanni/cartreader/files/6856009/N64.txt)
![N64 txt missing](https://user-images.githubusercontent.com/17235119/126495064-252d1635-6f29-448a-80c9-fd1985a41d4e.jpg)
![switch settings](https://user-images.githubusercontent.com/17235119/126495096-05a50ff6-3d6a-4671-89ce-e2475f51b63f.jpg)

**Comment by sanni** [comment]
Even though it complains about **N**64.txt missing what it really wants is an **n**64.txt. 

![jdtgdashdf](https://user-images.githubusercontent.com/19169494/126504541-7cae153f-05a4-4472-ba7b-e98e09694578.jpg)

**Comment by rrig** [comment]
Thanks for the quick reply, downloaded the master zip file, copy and pasted the sd code but still get the “N64.txt missing”.

Sent from Mail for Windows 10

**Reply by PsyK0p4T** [reply]
Maybe a dumb question but... did you put the n64.txt (and other files) at the root of your sd cart ? and not the folder itself.

Do you have similar message when you try other consoles using txt files ? like snes, gba, etc.

**Comment by rrig** [comment]
Not dumb at all, I appreciate the help. I did copy just the files to the sd card (nothing else is on the card). I formatted the card and tried again, still get N64.txt missing. SNES is working, N64 and SNES is all I have carts for. I did not install the GBA slot. Thanks again for any insight.

Sent from Mail for Windows 10

**Reply by PsyK0p4T** [reply]
ok... well we can do a complete check of your setup to maybe find the issue. Can you confirm that: 
- your microSD is FAT32 formatted
- at the root of your SD you got all the files given in "sd" folder from this github (open the n64.txt file from your sd to confirm its content and that the copy worked fine)
- you have your cart reader updated to latest version (v6.4 currently)
- the different switches on cart reader are following the screenshots from n64 wiki pages
And it should do the job, eventually :-/

**Reply by rrig** [reply]
will do, i'll let you know how it goes. Thx again

**Comment by rrig** [comment]
Figured it out, actually read the user guide and seen the sd needed to be formatted to exFAT. I get a CRC error but choose the option to ignore. Read a couple of games and the roms are working. I think updating to v6.4 and formatting to exFAT did the trick. I really appreciate your help.

Sent from Mail for Windows 10
